CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## CHANGELOG
 
+### 4.0.0
+> Update the routes. The base route is now `/books-music-movies/recommendations/best-books/`. And `staff-picks`, `ya`, `childrens` are three lists on the same level of the structure.
+> Add the list selectors that help the user choose the list by season/month or audience.
+> Update an individual page for every single book for supporting legacy staff-picks links and homepage's staff-picks section. There won't be links to those pages from the main page.
```

Comment:
Maybe "Update the BookPage component to support legacy staff-picks direct links and homepage's staff-picks component as the modal replacement from the previous staff-picks application. No direct links to picks in lists is supported at this time."  ??

Reply:
sure !